python/jobs.py

```diff
@@ -56,6 +56,12 @@ def __init__(self):
         self.json_property(int, 'design_status')
         self.json_property(int, 'payment_status')
         self.json_property(int, 'shipping_status')
+        self.json_property(int, 'unread_notifications_count')
+        self.json_property(int, 'unread_job_info_notifications_count')
+        self.json_property(int, 'unread_job_drafting_notifications_count')
+        self.json_property(int, 'unread_job_production_notifications_count')
+        self.json_property(int, 'unread_job_shipping_notifications_count')
+        self.json_property(int, 'unread_job_invoicing_notifications_count')
```

Comment:
you should add a comment indicating that these attributes are not embedded by default. and perhaps also seperate them to below `completed` and `priority`. see the examples in users.py and notifications.py

Reply:
thanks.